packages/cli-repl/src/format-output.ts

```diff
@@ -0,0 +1,60 @@
+import util from 'util';
+
+/**
+ * Return the pretty string for the output.
+ *
+ * @param {any} value - The output value.
+ * @param {string} type - The shell api type if known or undefined.
+ *
+ * @returns {string} The output.
+ */
+export default function formatOutput(output: any, type?: string): string {
+  if (type === 'Cursor') {
```

Comment:
I like the idea of doing this style of comparison! I think if we ever want to do anything interactive, it'll be more or less straightforward this way.